edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java

```diff
@@ -280,7 +279,8 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 									logRejuvenatingProcessor.getLogInvsNotLoweredInCatch().size(),
 									logRejuvenatingProcessor.getLogInvsNotTransformedInIf().size(),
 									this.isUseLogCategory(), this.isUseLogCategoryWithConfig(),
-									this.isNotLowerLogLevel(), resultsTimeCollector.getCollectedTime());
+									this.isNotLowerLogLevel(), this.getValueOfCheckIfCondition(),
```

Comment:
This is incorrect. The `edu.cuny.hunter.log.evaluation.handlers.EvaluationHandler.checkIfCondtion` field should be queried, just like it is for `this.isNotLowerLogLevel()`.

Reply:
BTW, where is this field read?

Reply:
https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/blob/ac4629689008336621213d4bf3ee045388437e7c/edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java#L381

Reply:
Oh, there is a bug here.

Reply:
The variable name is corrected.